src/components/charts/scatter-plot/scatter-plot-component.jsx

```diff
@@ -271,7 +271,7 @@ function ScatterPlot({
               top: `${chartScale.yScale(yAxisValue)}px`,
             }}
           >
-            {getLocaleNumber(yAxisValue, locale)}
+            {getLocaleNumber(roundSPI(yAxisValue), locale)}
```

Comment:
We don't need getLocaleNumber here as it will be an integer

Reply:
Since we have the round in an external function to change it if necessary, it would not be risky to remove the getLocalNumber?